infrastructure/Makefile

```diff
@@ -4,6 +4,11 @@ export ALGORAND_DATA=`pwd`/node_data/PrimaryNode
 export SANDBOX_PATH = ~/.algorand-sandbox
 export REMOTE_REPO = https://github.com/algorand/sandbox.git
 
+setup-private-net: create-private-net start-private-net setup-master-account status-private-net
+_rm-private-net:
+	rm -rf ./node_data
+recreate-private-net: stop-private-net _rm-private-net setup-private-net
```

Comment:
we can use `delete-private-net` instead of ` _rm-private-net` here

Reply:
OK

Reply:
done